cluster/config-defaults.yaml

```diff
@@ -117,10 +117,12 @@ zmon_worker_cpu: "750m"
 zmon_worker_count: "16"
 {{end}}
 logging_watcher_mem: "200Mi"
-logging_scalyr_mem: "200Mi"
+logging_scalyr_mem: "175Mi"
+logging_slo_heartbeat_mem: "10Mi"
```

Comment:
could you also bump it to `25Mi` to not add confusion later on? :)

Reply:
Sure can do. Just wanted to avoid any further work in case you decrease the minimum to sth lower than 25Mi :smile: